parsl/providers/errors.py

```diff
@@ -46,18 +46,18 @@ def __str__(self):
 
 
 class SubmitException(ExecutionProviderException):
-    '''Raised by the submit() method of a provider if there is an error in launching a task.
+    '''Raised by the submit() method of a provider if there is an error in launching a job.
     '''
 
-    def __init__(self, task_name, message, stdout=None, stderr=None):
-        self.task_name = task_name
+    def __init__(self, job_name, message, stdout=None, stderr=None):
+        self.job_name = job_name
```

Comment:
This is effectively an API change.  Any sense in (for a time) adding a property for `.task_name`?  Something like:
```python
@property
def task_name(self) -> str:  # *is* it a string?  No typing in __init__ signature ...
    log.warning("[...deprecated; use .job_name instead...]")  # remove after now() + 6mo
    return self.job_name
```

I'm also fine "just doing it," but a suggestion.

Reply:
i added something like this in a subsequent commit